src/main/java/gov/nasa/jpl/mbee/mdk/mms/sync/mms/MMSDeltaProjectEventListenerAdapter.java

```diff
@@ -248,23 +248,17 @@ private void determineChangesUsingCommitResponse(File responseFile, Set<String>
                 int size = 0;
                 String commitSyncDirection = "";
                 for(ObjectNode jsonObject : commitObjects) {
-                    if(jsonObject.isArray() && jsonObject.size() > 0) {
-                        for(int i = 0; i < jsonObject.size(); i++) {
-                            JsonNode currentNode = jsonObject.get(i);
-                            JsonNode sourceField = currentNode.get(MDKConstants.SOURCE_FIELD);
-                            boolean isSyncingCommit = sourceField != null && sourceField.isTextual() && MDKConstants.MAGICDRAW_SOURCE_VALUE.equalsIgnoreCase(sourceField.asText());
-                            if(isSyncingCommit) {
-                                commitSyncDirection = "Removed";
-                            } else {
-                                commitSyncDirection = "Added";
-                            }
-
-                            size = validateJsonElementArray(currentNode, isSyncingCommit, lockedElementIds, size);
-                        }
-                    } else { // what do we throw here? and should we?
-                        throw new IllegalStateException();
+                    JsonNode currentNode = jsonObject;
```

Comment:
@Enquier This issue was already fixed internally, but we neglected to push it back.  This change is ok for us